crates/c-api/include/wasmtime.h

```diff
@@ -75,7 +72,7 @@ WASM_API_EXTERN bool wasmtime_wat2wasm(
 
 WASMTIME_DECLARE_OWN(linker)
 
-WASM_API_EXTERN own wasmtime_linker_t* wasmtime_linker_new(wasm_store_t* store);
+WASM_API_EXTERN own wasmtime_linker_t* wasmtime_linker_new(wasm_store_t* store, bool allow_shadowing);
```

Comment:
We might end up having a number of configuration options, so could this become a configuration method rather than a constructor parameter?

Reply:
I'll change it to a function.